docs/source/overview/getting-started/install-guide.rst

```diff
@@ -5,57 +5,52 @@ Installation Guide
 
 EvaDB provides couple different installation options to allow easy extension to rich functionalities. 
 
-Default
+Use pip
 -------
 
-By Default, EvaDB installs only the minimal requirements.
+EvaDB supports Python (versions >= 3.8). We recommend installing with `pip` within an `isolated virtual environment <https://docs.python-guide.org/dev/virtualenvs/>`_.
 
-.. code-block::
+.. code-block:: bash
 
+    python -m venv evadb-venv
+    source evadb-venv/bin/activate
+    pip install --upgrade pip
     pip install evadb
 
-Vision Capability
------------------
+Install additional packages
+---------------------------
 
-You can install EvaDB with the vision extension. 
-With vision extension, you can run queries to do image classification, object detection, and emotion analysis workloads, etc.
+* `evadb[vision]` for vision dependencies. With vision dependencies, we can run queries to do image classification, object detection, and emotion analysis workloads, etc.
```

Comment:
Would it be better to use ` ``evadb[vision]`` `. Single ` (italic) is hard to see in the doc.